### PR TITLE
Add more unit tests

### PR DESCRIPTION
**Pull request by friedger**
This PR
* removes a duplicate error
* adds unit tests for some control branches
* increases code coverage to 67.4%

See https://github.com/Trust-Machines/stacks-sbtc/issues/646

### CODE REVIEW

File: sbtc-mini/contracts/sbtc-stacking-pool.clar [hunk 1]
```diff
@@ -82,9 +82,8 @@
 (define-constant err-vote-or (err u6040))
 (define-constant err-candidates-overflow (err u6041))
 (define-constant err-stacking-permission-denied (err u6042))
-(define-constant err-not-enough-locked-stx (err u6043))
```
Comment (friedger) [inline sbtc-mini/contracts/sbtc-stacking-pool.clar:85]: This is the same as err-not-enough-stacked

**Review by friedger — COMMENTED** [review]
:shipit:

File: sbtc-mini/tests/sbtc-stacking-pool_flow_test.clar [hunk 2]
```diff
@@ -57,10 +60,31 @@
 		(try! (check-sign-register))
         ;; @mine-blocks-before 1
 		(try! (check-sign-register-fail))
+        ;; @mine-blocks-before 1590
+        (try! (check-vote-too-early))
```
Comment (MarvinJanssen) [inline sbtc-mini/tests/sbtc-stacking-pool_flow_test.clar:64]: Nit: some of your files have mixed spaces and tabs.

**Review by MarvinJanssen — APPROVED** [review]
LGTM

File: sbtc-mini/tests/sbtc-stacking-pool_flow_test.clar (hunk 2 above)
Comment (friedger) [reply sbtc-mini/tests/sbtc-stacking-pool_flow_test.clar:64]: :-/

Where is clariform?

Comment (MarvinJanssen) [reply sbtc-mini/tests/sbtc-stacking-pool_flow_test.clar:64]: Sorry.

Comment (friedger) [reply sbtc-mini/tests/sbtc-stacking-pool_flow_test.clar]: Done

**Pull request merged**